app/controllers/posts_controller.rb

```diff
@@ -206,6 +207,7 @@ def expand_embed
 
   def recover
     post = find_post_from_params
+    RateLimiter.new(current_user, "delete_post", 3, 1.minute).performed! unless current_user.staff?
```

Comment:
I don't think we should set a rate limit for recover and instead allow users to recover immediately. If the `SiteSettings.post_edit_time_limit` happens to be smaller than 1 minute, I will end up with a scenario where I can no longer recover my post. WDYT? :smiley: